deployments/docker-compose/Dockerfile

```diff
@@ -4,4 +4,4 @@ COPY ./envoy.yaml /etc/envoy/envoy.yaml
 COPY ./cache_filter.wasm /usr/local/bin/cache_filter.wasm
 COPY ./singleton_service.wasm /usr/local/bin/singleton_service.wasm
 RUN chmod go+r /etc/envoy/envoy.yaml /usr/local/bin/cache_filter.wasm /usr/local/bin/singleton_service.wasm
-CMD /usr/local/bin/envoy -c /etc/envoy/envoy.yaml
+CMD /usr/local/bin/envoy -c /etc/envoy/envoy.yaml -l debug
```

Comment:
Let's keep the default log levels as it and change them whenever we need them accordingly in development. (debug, trace and component level logging).

Reply:
It was not supposed to be pushed. fixed :)